docs/platform_matrix.html

```diff
@@ -163,6 +163,34 @@
     loadsave_meter:   yes,
   },
 
+  android:
+  {
+    platform:         defunct("android"),
+    description:      "Android",
+    architecture:     "ARMv7-a (ELF) <br/> AArch64 (ELF) <br /> i686 (ELF) <br /> x86_64 (ELF)",
+    endian:           "Little",
+    toolchain:        "clang 8.0.2 <br /> binutils 2.27.0.20170315 <br /> bionic",
+    packaged:         _FAULTY("TBD"),
```

Comment:
Is there a better value for this? Probably okay if there isn't.

Reply:
Yes! Since we now have a proper self-contained APK, the correct value is "yes" - plus some Makefile fiddling.